.vale.ini

```diff
@@ -2,20 +2,14 @@
 StylesPath = .github/styles
 MinAlertLevel = suggestion
 
-Packages = Google
+# The docs/writing-rules.zip is generated by `pnpm docs:zipRules`
+Packages = Google, docs/writing-rules.zip
```

Comment:
Is it possible to embed the Google package in docs/writing-rules.zip?

Reply:
> Simplifying the .vale.ini to just importing this zip?

Yes, making the setup on dependent repos as small as possible

Reply:
Yes, we started with the basics Google rules we wanted to pick.

I prefer to keep them distinct, such that in `material-ui` repo we store the rules, and then each sub repo are able to configure its vale as they want by allowing or not different rules

Reply:
> I prefer to keep them distinct, such that in material-ui repo we store the rules, and then each sub repo are able to configure its vale as they want by allowing or not different rules

Is it possible for material-ui repo to provide the Google defaults and X/Toolpad override where necessary? That would allow @samuelsycamore to maintain the these defaults in a single location as well.

Reply:
Ok, I created https://github.com/mui/material-ui/issues/41225 to don't forget. I will take care of it after cleaning all the PR already open :)